src/cudafeatbin/compute-online-feats-cuda.cc

```diff
@@ -0,0 +1,123 @@
+// cudafeatbin/compute-online-feats-cuda.cc
+//
```

Comment:
How about putting this and your other compute-online-feats binary as test binaries in cudafeat/, and making it so they exit with status 0 instead of status one when called with no args?  Then they won't lengthen the non-test compile time unnecessarily.

Reply:
so the idea would be these would only be built when make test is called?  That seems reasonable.

Reply:
actually i think this binary would be useful outside of testing.  For people using the multi-process workflow this binary could replace a lot separate commands to compute online features.

Reply:
For us the feature processing is usually I/O bound, not compute bound.  Are you sure there are scenarios where this would make sense, once you consider the overhead of file I/O and so on?

Reply:
Certainly we see a benefit when fully integrated.  This suggests we are not I/O bound on input but perhaps we would be on output.   You will notice the timing above suggests some utility:

Libri-speech using the CPU binary: 1m 17 seconds.
Libri-speech using the GPU binary: 7.6

I'd be surprised that wav data input becomes I/O bound especially if reading from something like NVRAM.  Certainly if you are running off of NFS you could become I/O bound but I think for professional deployments that bottleneck could be easily mitigated.